lib/executor.js

```diff
@@ -47,6 +49,13 @@ TaskExec.prototype.process = function process(triggerOrAction, payload, cfg, sna
     }
 };
 
+/**
+ * Returns flow variables or empty object
+ * @returns {Object<String, String>}
+ */
+TaskExec.prototype.getFlowVariables = function getFlowVariables() {
```

Comment:
Core solution: make possible to fetch additional data from `this` in component code using dedicated method.
Suppose that this should be strategy for all similar future cases.